rotkehlchen/globaldb/handler.py

```diff
@@ -1203,6 +1227,144 @@ def get_historical_price_data(source: HistoricalPriceOracle) -> List[Dict[str, A
              'to_timestamp': entry[3],
              } for entry in query]
 
+    @staticmethod
```

Comment:
This function is not use anywhere. Why? I said the user should be given a choice what kind of restoring they have to do. Either completely delete everything, even used ones WITH BIG HUGE RED WARNING and possibly a confirmation popup. Also definitely hidden somewhere in the settings under the data settings and not somewhere easy to press by mistake.

Reply:
Added a new option to call this method. In case of conflicts the frontend prompts 2 warnings before actually doing the opeartion